lib/ims/lti/services/message_authenticator.rb

```diff
@@ -33,8 +33,7 @@ def simple_oauth_header
           @options.merge(
             {
               consumer_key: consumer_key,
-              consumer_secret: @secret,
-              callback: 'about:blank'
```

Comment:
lets hold off for a bit, I need to do some more testing.  It should be included automatically if it's in the launch_params, I just need to make sure that happens.

The issue is that it adds it even if it's not in the params, which causes the signature to break.  If it is in the params then everything works correctly.